lambdas/tests/unit/services/security/jwe-decrypter.test.ts

```diff
@@ -63,16 +65,26 @@ describe("JweDecrypter", () => {
         expect(result).toEqual(Buffer.from("decrypted content"));
     });
 
+    it("throws error when decrypts JWE fails", async () => {
+        try {
+            await jweDecrypter.decryptJwe(compactJwe);
+        } catch (error) {
```

Comment:
Typically with these I prefer to do something like
`expect(() => jweDecrypter.decryptJwe(compactJwe)).toThrow(JweDecrypterError("..."))`
because if the error never throws the test will just end and not throw a failure

Reply:
Yeah, I used that in one of the test, but I find they are sometime limited. In this case I wanted to do a `contains` part of the error message. Generally, I don't like the try catch approach but I opted for them if it is easier or makes the test a little clearier on the asserts

Reply:
That's fair - it is really clear. The expects do stack so you can do things like `.toThrow(expect.objectContaining({...}))` if you just want to test part of it? Alternately could you just add an `expect.assertions(2)` at the end of each test just to check nothing is getting missed off